validate.py

```diff
@@ -8,7 +8,9 @@
 def print_points():
   print('Points:')
   for user, user_points in util.get_user_points().items():
-    print('  %s: %s' % (user, user_points))
+    print('  %s: %s' % (user, user_points['total']))
+    for reason in [reason for reason in user_points if reason != 'total']:
```

Comment:
What about:

```
for reason, points_count in points.items():
  if reason == 'total':
    continue
  ...
```

Reply:
I was overthinking it, and ended up writing something that looks awful :)

Reply:
I think this is worth fixing

Reply:
Ack, thought I'd pushed. Yes, I agree, it looks like blended garbage.